components/list/list-item-drag-drop-mixin.js

```diff
@@ -339,6 +339,8 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 					opacity: 0;
 				}
 				:host([selected]) d2l-list-item-drag-handle,
+				:host([current]) d2l-list-item-drag-handle,
+				:host([_focusing-elem]) d2l-list-item-drag-handle,
```

Comment:
`_focusing-elem` and `current` only apply to the new items so this is not flagged.